src/sql/azdata.proposed.d.ts

```diff
@@ -670,10 +670,36 @@ declare module 'azdata' {
 		 */
 		delete?: boolean;
 	}
+
 	export interface DiffEditorComponent {
 		/**
 		 * Title of editor
 		 */
 		title: string;
 	}
+
+	export interface TextColumnOption {
+		content?: string
+	}
+
+	export interface RowDetailsOptions {
```

Comment:
Please add doc comments to all new properties/interface definitions (if you're extending an existing interface no need to add the comment in that case - but do so for all new properties in it)